### PR TITLE
Fix set command

### PR DESCRIPTION
**Pull request by daemonsy**
## What's up? 

`hubot heroku config:set xyz-app ABC=example.com` breaks because the regex matcher is too simplistic. 

This updates it to accept the common type of values, such as UUIDs, URIs, sentences. 

I got a small snag when trying to deal with naked `\n`. Any help is greatly appreciated.

Derived from From https://github.com/daemonsy/hubot-heroku/pull/8

### CODE REVIEW

File: test/fixtures/config-set.json [hunk 1]
```diff
@@ -1 +1,8 @@
-{ "CLOAK_ID": "10" }
+{
+  "CLOAK_ID"               : "example.com",
+  "UUID"                   : "d5126f0d-b3be-46af-883f-b330a73964f9",
```
Comment (daemonsy) [inline test/fixtures/config-set.json]: Don't like that the key-values have to be repeated again in the tests. Will consider refactoring later.

File: test/heroku-commands-spec.coffee [hunk 2]
```diff
@@ -152,20 +153,73 @@ describe "Heroku Commands", ->
...
+      room.user.say "Damon", "hubot heroku config:set shield-global-watch SENTENCE=\"Don't stop believin.\""
 
       setTimeout(->
-        expect(room.messages[1][1]).to.equal("@Damon Setting config CLOAK_ID => 10")
-        expect(room.messages[2][1]).to.equal("@Damon Heroku: CLOAK_ID is set to 10")
+        expect(room.messages[2][1]).to.equal("@Damon Heroku: SENTENCE is set to Don\'t stop believin.")
         done()
       , duration)
 
+    xit "handles RSA secret keys", (done) ->
```
Comment (daemonsy) [inline test/heroku-commands-spec.coffee]: Couldn't deal with naked `\n`, need someone with great regex foo like @arkham

**Comment by daemonsy** [comment]
Fixes https://github.com/daemonsy/hubot-heroku/issues/10

File: test/heroku-commands-spec.coffee (hunk 2 above)
Comment (Arkham) [inline test/heroku-commands-spec.coffee]: I think this works: `/heroku config:set (.*) (\w+)=('([\s\S]+)'|"([\s\S]+)"|([\s\S]+\b))/im`

Comment (daemonsy) [inline test/heroku-commands-spec.coffee]: How about a PR? :smile_cat:

Comment (Arkham) [inline test/heroku-commands-spec.coffee]: image

Hrm. Okay :angel:

**Pull request merged**